recipes/default.rb

```diff
@@ -26,4 +26,4 @@
   sysctl_param k do
     value v
   end
-end if node['sysctl']['params']
+end if node['sysctl'].attribute?('params')
```

Comment:
Hi, thanks for this update and your reactivity!! But the problem is still present (the problem occur when the node definition doesn't contain any sysctl configs (normal/node/…))

```chef
================================================================================
Recipe Compile Error in /var/chef/cache/cookbooks/base-system/recipes/default.rb
================================================================================

NoMethodError
-------------
undefined method `attribute?' for nil:NilClass

Cookbook Trace:
---------------
  /var/chef/cache/cookbooks/sysctl/recipes/default.rb:29:in `from_file'
  /var/chef/cache/cookbooks/base-system/recipes/default.rb:29:in `from_file'

Relevant File Content:
----------------------
/var/chef/cache/cookbooks/sysctl/recipes/default.rb:

 22:  ::Chef::Recipe.send(:include, SysctlCookbook::SysctlHelpers::Param)
 23:  
 24:  coerce_attributes(node['sysctl']['params']).each do |x|
 25:    k, v = x.split('=')
 26:    sysctl_param k do
 27:      value v
 28:    end
 29>> end if node['sysctl'].attribute?('params')
 30:  

Platform:
---------
x86_64-linux
```